planning/behavior_path_planner/autoware_behavior_path_start_planner_module/include/autoware/behavior_path_start_planner_module/start_planner_module.hpp

```diff
@@ -230,7 +230,7 @@ class StartPlannerModule : public SceneModuleInterface
   bool receivedNewRoute() const;
 
   bool isModuleRunning() const;
-  bool isCurrentPoseOnMiddleOfTheRoad() const;
+  bool isCurrentPoseOnCenterline() const;
```

Comment:
Can you apply the same change of `EgoCenterline`?

Reply:
@takayuki5168 

Addressed in the last commit